client/src/components/dashboard/Sidebar.tsx

```
@@ -0,0 +1,75 @@
+            <li><Link to="/dashboard/edit/profile" className="block p-2 hover:bg-purple-500 rounded font-semibold text-xl text-center border border-white bg-purple-700">Edit Profile</Link></li>
+            <li><Link to="/dashboard/analytics" className="block p-2 hover:bg-purple-500 rounded font-semibold text-xl text-center border border-white bg-purple-700">Analytics</Link></li>
+            <li><Link to="/dashboard/settings" className="block p-2 hover:bg-purple-500 rounded font-semibold text-xl text-center border border-white bg-purple-700">Settings</Link></li>
+          </ul>
+        </nav>
+      </div>
+      
+
+      <div className="md:hidden fixed top-38 left-2 z-50 flex items-center">
+  {/* Curved Background */}
+  {!isOpen && (
+    <div className="absolute -right-4 top-1/2 -translate-y-1/2 w-28 h-16 bg-purple-900 rounded-r-full rounded-l-[40px] shadow-lg"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)